src/main/java/reposense/util/TimeUtil.java

```diff
@@ -17,23 +16,31 @@ public static void startTimer() {
     /**
      * Returns the formatted elapsed time from {@code startTime} until current time
      */
-    public static String getElapsedTimeMessage() {
+    public static String getElapsedTime() {
         long endTime = System.nanoTime();
         double elapsedTime = (double) (endTime - startTime) / 1_000_000_000.0;
         int elapsedHours = (int) elapsedTime / 3600;
         int elapsedMinutes = (int) (elapsedTime % 3600) / 60;
         double elapsedSeconds = elapsedTime % 60;
+        String formattedElapsedTime = "";
 
         if (elapsedHours > 0) {
-            elapsedTimeMessage += String.format(" %d hour(s)", elapsedHours);
```

Comment:
why are the spaces removed as well?

Reply:
My apologies. I mistakenly thought the original code had some redundant spaces.
I have reverted this to the original style.